public/v4/apps/whisper-asr-webservice.yml

```diff
@@ -0,0 +1,36 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: onerahmet/openai-whisper-asr-webservice:$$cap_app_version
+        volumes:
+            - $$cap_appname-data:/var/lib/postgresql/data
+        restart: always
+        environment:
```

Comment:
Aren't these needed?
```
ASR_MODEL=base -e ASR_ENGINE=openai_whisper
```

Reply:
it work without but yes It's better with those
sorry this pr has been open too fast